drivers/overlay/mount.go

```diff
@@ -8,21 +8,33 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"path/filepath"
 	"runtime"
+	"strings"
 
 	"github.com/containers/storage/pkg/reexec"
 	"golang.org/x/sys/unix"
 )
 
 func init() {
-	reexec.Register("storage-mountfrom", mountFromMain)
+	reexec.Register("storage-mountfrom", mountOverlayFromMain)
 }
 
 func fatal(err error) {
 	fmt.Fprint(os.Stderr, err)
 	os.Exit(1)
 }
 
+func bindMount(source, destination, data string) (string, error) {
```

Comment:
`pkg/mount` already has generic mount function, could we reuse it ?

Reply:
Dropping the function.